htdocs/product/card.php

```diff
@@ -2099,7 +2099,7 @@
 
 				print '<tr><td>'.$langs->trans('DefaultUnitToShow').'</td><td>';
 				if ($unit !== '') {
-					print $langs->trans($unit);
+					print $unit;
```

Comment:
According to documentation of getLabelOfUnit, the result must then be translated with lang->trans before being rendered.